sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -971,9 +977,25 @@ void CAFMaker::produce(art::Event& evt) noexcept {
       FindManyPStrict<anab::Calorimetry>(slcTracks, evt,
            fParams.TrackCaloLabel() + slice_tag_suff);
 
-    art::FindManyP<anab::ParticleID> fmPID =
+    art::FindManyP<anab::ParticleID> fmChi2PID =
       FindManyPStrict<anab::ParticleID>(slcTracks, evt,
-          fParams.TrackPidLabel() + slice_tag_suff);
+          fParams.TrackChi2PidLabel() + slice_tag_suff);
+
+    art::FindManyP<sbn::ScatterDCA> fmScatterDCA =
+      FindManyPStrict<sbn::ScatterDCA>(slcTracks, evt,
+          fParams.TrackScatterDCALabel() + slice_tag_suff);
+
+    art::FindManyP<sbn::StoppingChi2Fit> fmStoppingChi2Fit =
+      FindManyPStrict<sbn::StoppingChi2Fit>(slcTracks, evt,
+          fParams.TrackStoppingChi2FitLabel() + slice_tag_suff);
+
+    art::FindManyP<sbn::MVAPID> fmTrackDazzle =
+      FindManyPStrict<sbn::MVAPID>(slcTracks, evt,
+          fParams.TrackDazzleLabel() + slice_tag_suff);
+
+    art::FindManyP<sbn::MVAPID> fmShowerRazzle =
+      FindManyPStrict<sbn::MVAPID>(slcShowers, evt,
+          fParams.ShowerRazzleLabel() + slice_tag_suff);
```

Comment:
We probably need to factorize CAFMaker some more, since `produce()` keeps growing and growing. But that's not directly related to this PR, just that you're adding 4 new classes of things to fill to the N that are already there.

Reply:
I agree a reorganisation would be good, but beyond the scope of this PR